bcl2fastq/index_summary.py

```diff
@@ -153,7 +159,8 @@ def update_pruned_unknown_hidden(run_alias):
 
 @app.callback(
     dep.Output('known_index_bar', 'figure'),
-    [dep.Input('sample_run_hidden', 'children')]
+    [dep.Input('sample_run_hidden', 'children')
+     ]
```

Comment:
Inconsistent indentation.

Reply:
Please reformat the code before each commit. In PyCharm this is done with Code > Reformat Code, or `Ctrl+Alt+L`. You can also install the git pre-commit hook which will do it for you.